client/packages/dashboard/src/widgets/DistributionWidget.tsx

```diff
@@ -10,10 +10,12 @@ import {
   FnUtils,
   useToggle,
   ApiException,
+  RouteBuilder,
 } from '@openmsupply-client/common';
 import { useFormatNumber, useTranslation } from '@common/intl';
 import { useDashboard } from '../api';
 import { useOutbound } from '@openmsupply-client/invoices';
+import { AppRoute } from 'packages/config/src';
```

Comment:
I think we want to stick to importing from the top-level packages, so this one would be:
```
import { AppRoute } from '@openmsupply-client/config';
```

Same for the other Widgets in here